xbmc/utils/SysfsUtils.cpp

```diff
@@ -16,105 +16,140 @@
 #include <fcntl.h>
 #include <string.h>
```

Comment:
Update the headers, most are unused

I think you would need
`<sys/stat.h>`, `<fcntl.h>`, and `<unistd.h>` (according to open(3p) and write(3p))